Mechatronics.tex

```diff
@@ -506,19 +506,19 @@ \subsection{Other Properties}
 	\end{figure} 
  
  \begin{asparaenum}[a)]
- 	\item Simple \textit{bends} can be made on a linear portion of the flat pattern. This can be done with a finger bender, press brake, or (in a pinch) a vise with a hammer.
+ 	\item Simple \textit{bends} can be made on a linear portion of the flat pattern. This can be done with a finger bender, press brake, or, in a pinch, a vise with a hammer.
  	\item Flat portions of material can be \textit{rolled} into an arc, or even full circle.
  	\item \textit{Bead rolling} can be done on any portion of a part to provide additional stiffness.	
  	\item \textit{Hemming} provides a smooth, radiused outer corner and some additional stiffness. First, a bend is made, and then it is bent all the way to 180 degrees. This can be done with a finger bender or press brake, and additional clamping to finish the hem.
  	\end{asparaenum}
  	
- 	Most CAD packages have tools to design parts made with sheetmetal. You can draw up the bent part, and then the CAD package will determine how to 'unfold' and produce a flat pattern that can be cut out.
+ 	Most CAD packages have tools to design parts made with sheet metal. You can draw up the bent part, and then the CAD package will determine how to 'unfold' and produce a flat pattern that can be cut out.
```

Comment:
I don't like that sheetmetal isn't actually a word. This upsets me.

Also get rid of quotes on unfold.